### PR TITLE
Updated the docs, though I'm not sure it's used much anymore.

### PR DESCRIPTION
**Pull request by bennyvasquez**
docs/assembling_twir.md was pretty out of date. Updated it mostly with educated guesses.

### CODE REVIEW

File: docs/assembling_twir.md [hunk 1]
```diff
@@ -54,12 +52,12 @@ These are collected using GitHub queries:
 
 ## Events
 
-[@nellshamrell](https://github.com/nellshamrell) currently covers this section, but would welcome help from additional editors!
+[@andrewpollack](https://github.com/andrewpollack) currently covers this section, but would welcome help from additional editors!
```
Comment (mariannegoldin) [inline docs/assembling_twir.md:55]:
```suggestion
[@andrewpollack](https://github.com/andrewpollack) and @mariannegoldin](https://github.com/mariannegoldin) currently cover this section, but would welcome help from additional editors!
```

Comment (andrewpollack) [reply docs/assembling_twir.md]: Absolutely! Merging this commit (with a minor edit on the suggestion formatting) :raised_hands:

File: docs/assembling_twir.md [hunk 2]
```diff
@@ -6,9 +6,7 @@ This documents the general process for assembling each of the sections.
 
 ## Updates from the Rust Community
 
-[@nellshamrell](https://github.com/nellshamrell) currently covers this section, but would welcome help from additional editors!
-
-(Thank you to [@cdmistman](https://github.com/cdmistman) for covering this section so long! We look forward to when you can return!)
+The full editoral staff works together to cover this section, but the primary contributors are [@extrawurst](https://github.com/extrawurst), [@ericseppanen](https://github.com/ericseppanen), and [@cdmistman](https://github.com/cdmistman).
```
Comment (mariannegoldin) [inline docs/assembling_twir.md:9]: Would it make sense to note that @nellshamrell provides lead editorial guidance for the overarching process?

Comment (bennyvasquez) [reply docs/assembling_twir.md:9]: yes! That absolutely does. Want to add something in there, or want me to?

Comment (mariannegoldin) [reply docs/assembling_twir.md:9]: I'll defer to your judgment :)

**Comment by bennyvasquez** [comment]
This branch was out of date and I... ended up in git hell. Creating a new PR from a fresh branch here: https://github.com/rust-lang/this-week-in-rust/pull/3822

**Pull request closed without merging**